internal/applicationsnapshot/input.go

```diff
@@ -81,37 +82,46 @@ func DetermineInputSpec(ctx context.Context, input Input) (*app.SnapshotSpec, er
 	var snapshot snapshot
 	provided := false
 
+	if input.SnapshotSource != "" {
+		var content []byte
+		var err error
+		fs := utils.FS(ctx)
+		content, err = afero.ReadFile(fs, input.SnapshotSource)
+		if err != nil {
+			log.Debugf("could not read --snapshot-source as a file: %v", err)
```

Comment:
The actual name of the parameter is not known by this package. Consider removing it from the error message.